crates/ironrdp-graphics/src/pointer.rs

```diff
@@ -38,7 +38,7 @@ pub enum PointerError {
 }
 
 /// Represents RDP pointer in decoded form (color channels stored as RGBA pre-multiplied values)
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
suggestion: In this case, deriving Clone is not necessary anymore. If you really want to derive this, please open a separate PR.